src/first-degree-equation.js

```diff
@@ -0,0 +1,19 @@
+function solveFirstDegreeEquation(number1, number2) {
+  let result = '';
+  let temp = 0;
+  if (number1 === 0 && number2 === 0) {
+    return result;
```

Comment:
should return `the equation has infinitely many solutions` instead of empty string

Reply:
so, return a string "the equation has infinitely many solutions", right?

Reply:
right